standards/qa-process.md

```diff
@@ -50,6 +53,7 @@ a. PM sends an email to qa@shift3tech.com. Answer the following questions:
 2. Team is assigned to the project repo on Github in order to post issues and coordinate with developers.
 3. If applicable, review for each user types abilities and limitations.
 4. Credentials are given to the team.
+5. If QA is billable to the client, such as for T&M projects, add the QA team to the Tick project.
```

Comment:
I would clarify as follows: `the PM will add the QA team to the Tick project.`

Reply:
Agreed, developers cannot do this